Tyro/Error.swift

```diff
@@ -8,26 +8,26 @@
 
 import Foundation
 
-public protocol JSONErrorType : ErrorType {}
+public protocol JSONErrorType : Error {}
 
 public enum JSONError : JSONErrorType {
     case Array([JSONError])           // array of errors
     case TypeMismatch(String, String) // expected / actual
-    case Error(ErrorType, String)     // error / message
+    case Error(Error, String)     // error / message
```

Comment:
Canonical case names for Swift 3 structures begin with a lowercase letter.  That means `Error` should not be a case name.